.buildkite/scripts/lucene-snapshot/get-credentials.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# WARNING: this script will echo the credentials to the console. It is meant to be called from another script and captured in a variable.
+# It should really only be used inside .buildkite/hooks/pre-command
+
+VAULT_ROLE_ID=$(vault read -field=role-id secret/ci/elastic-elasticsearch/legacy-vault-credentials)
```

Comment:
Is this a side effect of us building some repo we don't "own"? Can we not migrate these creds to the regular buildkite vault instance?

Reply:
It's not a set of credentials, it's actually a backend that generates credentials in AWS on-the-fly. I was blocked for like two weeks trying to get it migrated with the CI team, and just moved on